File: mobile/Emotracker/Emotracker.Core/Model/TokenDTO.cs
using System;

namespace Emotracker.Core
{
	[Serializable]
	public class TokenDTO
	{
		public string Id { get; set; }
		public string Key { get; set; }
		public string Token { get; set; }
		public string Valid { get; set; }
	}
}

File: server/emotracker-core/src/main/java/com/innerman/emotracker/core/dto/TokenDTO.java
package com.innerman.emotracker.core.dto;

import org.hibernate.validator.constraints.NotBlank;

import javax.validation.constraints.NotNull;
import java.io.Serializable;

/**
 * Created by petrpopov on 01.03.14.
 */
public class TokenDTO implements Serializable {

    @NotNull
    @NotBlank
    private String key;

    @NotNull
    @NotBlank
    private String token;

    public String getKey() {
        return key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }
}
